servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/DefaultHttpLoadBalancerProviderTest.java

```diff
@@ -41,46 +40,46 @@ void cleanup() {
     }
 
     @Test
-    void defaultsToNoHost() throws Exception {
+    void defaultsToNoHost() {
         SingleAddressHttpClientBuilder<HostAndPort, InetSocketAddress> builder =
                 HttpClients.forSingleAddress("testhost", 80);
-        assertThat(builder, not(instanceOf(DefaultHttpLoadBalancerProvider.LoadBalancerIgnoringBuilder.class)));
+        assertThat(builder, instanceOf(DefaultSingleAddressHttpClientBuilder.class));
```

Comment:
Are these changes intentional? Why some tests change it but others didn't?

Reply:
They are intentional changes: I couldn't test for the exact class before because they are package private to the `http.netty` package but now they're in scope. Requiring the exact class is a tighter requirement/better test since in theory we could have yet a second wrapper which would allow the new builder to function, but would pass the test.